tests/test_ios_testflight_external_distribution.py

```
@@ -89,6 +89,50 @@ def main():
     else:
         _check(False, "missing external group fails loudly")
 
+    submissions = []
+
+    def fake_submit(token, build_id):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `fake_submit`

Add return type annotation: `None`

(ANN202)